### PR TITLE
Add Subscription-details page, Download license, Delete subscription

### PR DESCRIPTION
**Pull request by sajibcse68**

### CODE REVIEW

File: js/routers/router.js [hunk 1]
```diff
@@ -22,19 +22,21 @@ module.exports = Backbone.Router.extend({
     'invoice': 'invoiceList',
     'invoice/:cpid': 'invoiceDetails',
     'invoice/:name/print': 'printInvoice',
+    'purchage': 'purchageSubscription',
+    ':sku?*query': 'subscriptionDetails',
```
Comment (a8uhnf) [inline js/routers/router.js]: **suggestion:**
* may be `?*query` is unnecessary in route. query strings automatically passing in the method

File: js/routers/router.js [hunk 2]
```diff
@@ -73,9 +75,36 @@ module.exports = Backbone.Router.extend({
...
+  purchageSubscription() {
+    global.clearAllHtmlContent();
+    this.closeReuseObjects();
+  },
   closeReuseObjects() {
     if (!_.isUndefined(this.mainView)) {
       this.mainView.close();
     }
+  },
+  parseQueryString(queryString) {
```
Comment (TamalSaha) [inline js/routers/router.js]: Why do we need this?

File: js/routers/router.js (hunk 1 above)
Comment (sajibcse68) [inline js/routers/router.js]: Yes, it is automatically passed. `?*query` is used for more readable code :)

File: js/routers/router.js (hunk 2 above)
Comment (sajibcse68) [inline js/routers/router.js]: Necessary for `key: value` parsing of whole query.
Say, query = `?a1=v1&a2=v2`
this method output: `queryObj = {a1: v1, a2: v2}`

**Pull request merged**